doc/REST-interface.md

```diff
@@ -57,25 +59,25 @@ Example:
 ```
 $ curl localhost:18332/rest/getutxos/checkmempool/b2cdfd7b89def827ff8af7cd9bff7627ff72e5e8b0f71210f92ea7a4000c5d75-0.json 2>/dev/null | json_pp
 {
-   "chaintipHash" : "00000000fb01a7f3745a717f8caebee056c484e6e0bfe4a9591c235bb70506fb",
    "chainHeight" : 325347,
+   "chaintipHash" : "00000000fb01a7f3745a717f8caebee056c484e6e0bfe4a9591c235bb70506fb",
+   "bitmap": "1",
    "utxos" : [
       {
+         "txvers" : 1
+         "height" : 2147483647,
+         "value" : 8.8687,		 
          "scriptPubKey" : {
-            "addresses" : [
-               "mi7as51dvLJsizWnTMurtRmrP8hG2m1XvD"
-            ],
-            "type" : "pubkeyhash",
+            "asm" : "OP_DUP OP_HASH160 1c7cebb529b86a04c683dfa87be49de35bcf589e OP_EQUALVERIFY OP_CHECKSIG",
             "hex" : "76a9141c7cebb529b86a04c683dfa87be49de35bcf589e88ac",
             "reqSigs" : 1,
-            "asm" : "OP_DUP OP_HASH160 1c7cebb529b86a04c683dfa87be49de35bcf589e OP_EQUALVERIFY OP_CHECKSIG"
-         },
-         "value" : 8.8687,
-         "height" : 2147483647,
-         "txvers" : 1
+            "type" : "pubkeyhash",
+            "addresses" : [
+               "mi7as51dvLJsizWnTMurtRmrP8hG2m1XvD"
```

Comment:
should probably dashify this

Reply:
what type of address should it be? :see_no_evil: I'm not too familiar with bitcoin's address types

Reply:
that's a testnet/regtest p2pkh

Reply:
Actually, this file need to be dashified as a whole. Let's ignore it for now then.